eclass/kernel-install.eclass

```diff
@@ -607,6 +607,16 @@ kernel-install_pkg_preinst() {
 	[[ ! -d ${kernel_dir} ]] &&
 		die "Kernel directory ${kernel_dir} not installed!"
 
+	# We moved this in order to omit it from the binpkg, move it back
+	if [[ -r "${T}/signing_key.pem" ]]; then
+		# cp instead of mv to set owner to root in one go
+		(
+			umask 066 &&
+			cp "${T}/signing_key.pem" "${kernel_dir}/certs/signing_key.pem" ||
+				die
```

Comment:
Technically, you could move the `die` out of the subshell, i.e. `) || die`.

Reply:
That does look a bit nicer, Thanks!

Reply:
Yeah, it seems more reasonable with the `^^` chaining inside. It's actually something from old EAPIs, when `die` wouldn't be guaranteed to work inside a subshell.